infra/terraform/modules/team/main.tf

```
@@ -14,6 +14,11 @@
  * limitations under the License.
  */
 
+locals {
+  teams = distinct(flatten([for value in var.repos_map : value.groups if try(value.groups, null) != null]))
+  users = distinct(flatten([for value in var.repos_map : value.owners if try(value.groups, null) != null]))
```

Shouldn't this be owners?
```suggestion
  users = distinct(flatten([for value in var.repos_map : value.owners if try(value.owners, null) != null]))
```

---

Great Catch!